problemsets-mat223/modules/module3-exercises.tex

```diff
@@ -161,7 +161,7 @@
 				\end{enumerate}
 				\item
 				\ref{PROBSET3-R2-iv} $\Span\Set*{\mat{1\\-1}} + \Set*{\mat{1\\0}}$
-				\ref{PROBSET3-R2-v} $\Span\Set*{\mat{5\\3}} + \Set*{\mat{-4\\-3}}$
```

Comment:
Let's put a semicolon separating the two answers.

Reply:
Fixed!